packages/client/src/utils.ts

```diff
@@ -9,4 +9,43 @@
  * Copyright (C) The OpenCRVS Authors. OpenCRVS and the OpenCRVS
  * graphic logo are (registered/a) trademark(s) of Plan International.
  */
+import { storage } from '@client/storage'
+import { APPLICATION_VERSION } from '@client/utils/constants'
+import { IUserData } from '@client/declarations'
+
 export type Omit<T, K extends keyof T> = Pick<T, Exclude<keyof T, K>>
+
+export async function validateApplicationVersion() {
+  const appVer = localStorage.getItem('application-version')
+
+  if (
+    !appVer ||
+    (appVer !== APPLICATION_VERSION &&
+      isNewAppVersion(appVer, APPLICATION_VERSION))
+  ) {
+    localStorage.setItem('application-version', APPLICATION_VERSION)
+    const userData = await storage.getItem('USER_DATA')
+    const allUserData: IUserData[] = !userData
+      ? []
+      : (JSON.parse(userData) as IUserData[])
+
+    allUserData.map((userData) => {
+      userData['declarations'] = []
+      return userData
+    })
+
+    await storage.setItem('USER_DATA', JSON.stringify(allUserData))
```

Comment:
We also need to replace the `running-version` with the current application version